gen3/bin/kube-setup-roles.sh

```diff
@@ -29,6 +29,13 @@ if [[ -z "$JENKINS_HOME" ]]; then
   if ! g3kubectl get rolebindings/devops-binding > /dev/null 2>&1; then
     g3kubectl apply -f "${GEN3_HOME}/kube/services/jenkins/rolebinding-devops.yaml"
   fi
+
+  if ! g3kubectl get serviceaccounts/jenkins-service > /dev/null 2>&1; then
+    g3kubectl apply -f "${GEN3_HOME}/kube/services/ssjdispatcher/serviceaccount.yaml"
+  fi
+  if ! g3kubectl get rolebindings/devops-binding > /dev/null 2>&1; then
```

Comment:
you should be testing for the `ssjdispatcher-binding`, and rename the file

Reply:
Tested and fixed in a55945ffa92f01cdd9dc040a8fa35656ee373220